tests/tests.go

```diff
@@ -676,6 +676,9 @@ func getAggregatedTaskStatus(issue *api.Issue) (api.TaskStatus, error) {
 
 // waitIssuePipeline waits for pipeline to finish and approves tasks when necessary.
 func (ctl *controller) waitIssuePipeline(id int) (api.TaskStatus, error) {
+	// Sleep for two seconds between issues so that we don't get migration version conflict because we are using second-level timestamp for the version string.
```

Comment:
```suggestion
	// Sleep for two seconds between issues so that we don't get migration version conflict because we are using second-level timestamp for the version string. We choose sleep because it mimics the user's behavior.
```